com.jetbrains.Toolbox.yml

```diff
@@ -0,0 +1,65 @@
+id: com.jetbrains.Toolbox
+command: jetbrains-toolbox
+runtime: org.freedesktop.Platform
+runtime-version: "24.08"
+sdk: org.freedesktop.Sdk
+tags:
+  - proprietary
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --device=dri
+  - --socket=x11
+  - --socket=ssh-auth
+  - --share=network
+  - --socket=session-bus
+  - --talk-name=org.freedesktop.secrets
+  - --talk-name=org.kde.StatusNotifierWatcher
+  - --filesystem=home
```

Comment:
why does it need entire home? where are they downloaded by default?

Reply:
So you need $HOME/.local/share/JetBrains only then. Config files are supposed to be in ~/.var/app for Flatpak apps